lib/github-auth.js

```diff
@@ -214,14 +214,15 @@ function rmGithubToken(token) {
 
 // Act like request(), but tweak headers and query to avoid hitting a rate
 // limit.
-function githubRequest(request, url, query, cb) {
+function githubRequest(request, url, query, cb, customHeaders = {}) {
   query = query || {};
   // A special User-Agent is required:
   // http://developer.github.com/v3/#user-agent-required
-  const headers = {
+  const defaultHeaders = {
     'User-Agent': 'Shields.io',
     'Accept': 'application/vnd.github.v3+json',
   };
+  const headers = Object.assign({}, defaultHeaders, customHeaders);
```

Comment:
Are these changes necessary?

Reply:
I wanted to pass custom header to this method. Licenses API requires this header https://developer.github.com/v3/licenses/. Without it I wouldn't get license file url.

Reply:
You don't seem to be doing that though. What am I missing?

Reply:
In `lib/suggest.js`, in function `githubLicense` I'm invoking `githubAuth.request` with 5 parameters and the last one (after callback) is Accept header.

Reply:
But maybe it is not required. Give me a moment...

Reply:
Ahhh I see, it's after the callback.

That's super confusing and too easy to miss. I'd probably suggest moving that parameter before the callback, and updating all the other calls to pass an empty object. That's consistent with the way we handle `query`.

Alternatively you could do something clever like `request`, which detects whether each parameter is an option or the callback. I think this is more trouble than it's worth, especially because I'm hoping we rewrite all this stuff using promises and async/await pretty soon.

Reply:
I know it's confusing, but I didn't want to change method parameters order (a lot of code/people using it). 
In the meantime, I removed this custom headers. With default media type in the Accept header API returns right data. 
Sorry for this changes.

Reply:
Even better!